dll/shellext/acppage/CLayerUIPropPage.cpp

```diff
@@ -40,6 +40,9 @@ static struct {
     { L"Windows 7 (SP1)", L"WIN7SP1" },
     { L"Windows 8", L"WIN8RTM" },
     { L"Windows 8.1", L"WIN81RTM" },
+    { L"Windows 10", L"WIN10RTM" },
+    { L"Windows Server 2016", L"WINSRV16RTM" },
+    { L"Windows Server 2019", L"WINSRV19RTM" },
```

Comment:
I am fine with this change, if you remove:
XP sp2, Vista, Vista SP1, and Windows 8 from this list.

Reply:
Mark:
Remove `8`: and/or `7`?
And rename `2000` to `2000 (SP4)`?

Reply:
My comment was pretty clear imo, no reason to add confusion.

Reply:
Removed.
Now that compatibility modes disappeared from accpage, but the default entry became Win2k3 SP1 instead of XP SP3, as before. Is it possible to change this manually? If yes, can I leave it as is? Or I necessarily need to change the default value? Thanks.
![compatibility_modes_ROS_updated](https://user-images.githubusercontent.com/26385117/67622656-ee2dc980-f824-11e9-9e3e-81bb4bf7eab9.png)

Reply:
in `CLayerUIPropPage::OnInitDialog`, change the line from `ComboBox_SetCurSel(cboMode, 5);` to `ComboBox_SetCurSel(cboMode, 4);`

Reply:
Fixed. Thanks.
Now default entry is XP SP3 again, same as before.
![compatibility_modes_updated](https://user-images.githubusercontent.com/26385117/67678867-5ea52980-f990-11e9-8d37-d9a4dcbda46f.png)